.github/actions/action-setup_task-installPyProject/action.yml

```diff
@@ -46,18 +73,38 @@ runs:
         virtualenvs-create: true
 
     # Setup & configure Python
+    - name: Create cache-key file
+      shell: bash
+      run: echo "${{ inputs.cache-key }}" >> .git/x-gh-cache-key
     - name: Setup python
       uses: actions/setup-python@v4
       id: setup-python
       with:
         python-version: ${{ inputs.python-version }}
         cache: "${{ inputs.manager }}"
+        cache-dependency-path: |
+          **/poetry.lock
```

Comment:
Might be pre-emptive, but wondering about replacing `**/poetry.lock` with `**/{{ inputs.manager }}.lock`. This change would presume other dependency managers also name their `.lock` files similarly, which may not be the case.

Reply:
I'd say this is pre-emptive, easy enough to switch when/if we're actually implementing these other managers